pkg/controller/migplan/validation.go

```diff
@@ -448,3 +466,38 @@ func (r ReconcileMigPlan) validateDestinationNamespaces(plan *migapi.MigPlan) er
 
 	return nil
 }
+
+// Validate PV actions.
+func (r ReconcileMigPlan) validatePvAction(plan *migapi.MigPlan) error {
+	list := []string{}
+	actions := map[string]bool{
+		migapi.PvCopyAction: true,
+		migapi.PvMoveAction: true,
+	}
+	for _, pv := range plan.Spec.PersistentVolumes.List {
+		_, found := actions[pv.Action]
+		if !found {
+			list = append(list, pv.Name)
```

Comment:
Looks like `list` will contain all PVs with _invalid_ actions set

Reply:
Renamed to `invalid`.